crates/amaru/src/consensus/header_validation.rs

```diff
@@ -1,7 +1,7 @@
 use crate::{consensus::ValidateHeaderEvent, ledger::kernel::epoch_from_slot, sync::PullEvent};
 use gasket::framework::*;
+use ouroboros::consensus::BlockValidator;
```

Comment:
We might consider some clippy rules to avoid those kind of changes, if that's necessary.

Reply:
Not sure I understand what you mean @jeluard? What kind of changes?

Reply:
The change in `use` ordering. Nitpick :)